.github/workflows/docker.yaml

```diff
@@ -51,6 +48,9 @@ jobs:
 
       - name: Login to DockerHub
         uses: docker/login-action@v2
+        if:
```

Comment:
i think this `if` clause isn't really necessary.

Reply:
It failed when I ran it from a `iameli-streams` fork.